sourced/ml/cmd_entries/repos2ids.py

```diff
@@ -0,0 +1,24 @@
+import logging
+from uuid import uuid4
+
+from sourced.ml.transformers import Ignition, ContentExtractor, ContentDeserializer, \
+    Content2Ids, HeadFiles, Cacher
+from sourced.ml.utils import create_engine, EngineConstants
+
+
+def repos2ids_entry(args):
+    log = logging.getLogger("repos2ids")
+    engine = create_engine("repos2ids-%s" % uuid4(), **args.__dict__)
+    documents_column_name = [EngineConstants.Columns.RepositoryId,
+                             EngineConstants.Columns.PathId]
+    ids_transformer = Content2Ids(args, documents_column_name)
+
+    pipeline = Ignition(engine) \
+        .link(HeadFiles()) \
+        .link(ContentExtractor()) \
+        .link(ContentDeserializer()) \
+        .link(ids_transformer)
+    ids = pipeline.execute()
+
+    log.info("Writing %s", args.output)
+    ids_transformer.save(ids)
```

Comment:
I would make it a static method and then `.link(Content2Ids(args, column_names))`